src/olympia/amo/tests/__init__.py

```diff
@@ -1075,3 +1077,13 @@ def prefix_indexes(config):
 
     settings.CACHE_PREFIX = 'amo:{0}:'.format(prefix)
     settings.KEY_PREFIX = settings.CACHE_PREFIX
+
+
+def reverse_ns(viewname, api_version=None, args=None, kwargs=None, **extra):
+    api_version = api_version or settings.REST_FRAMEWORK['DEFAULT_VERSION']
```

Comment:
It makes sense to use `rest_framework.settings:api_settings` here, e.g

```python
from from rest_framework.settings import api_settings
api_settings.DEFAULT_VERSION
```

which is the official interface to the `settings.REST_FRAMEWORK` object.

Reply:
https://github.com/mozilla/addons-server/pull/8557/commits/5b8de05f2f2d805a6d01410ed2f34456c3e72b38